### PR TITLE
bradl3yC - 3219 - Change heading levels per cognition rules

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
- [x] - Change H4s to H3s to align with cognition rules regarding proper nesting rules

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/hca/enrollment-status-helpers.jsx [hunk 1]
```diff
@@ -357,10 +357,10 @@ export function getFAQContent(enrollmentStatus) {
 
   const faqBlock1 = (
     <>
-      <h4>
+      <h3>
```
Comment (kelsonic) [inline src/applications/hca/enrollment-status-helpers.jsx]: Just want to confirm as I haven't tested this locally. Is the styling still the same after upgrading these `h4` tags to `h3`?

Comment (kelsonic) [reply src/applications/hca/enrollment-status-helpers.jsx]: You could add something like this to ensure that the `h3` tags still have the `h4` tag styling:

```jsx
<h3 className="vads-u-font-size--h4"></h3>
```

**Comment by bradl3yC** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review 👍

**Review by cvalarida — APPROVED** [review]

**Pull request merged**